src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -732,7 +732,8 @@ namespace winrt::TerminalApp::implementation
     void TerminalPage::_OpenNewTab(const NewTerminalArgs& newTerminalArgs)
     try
     {
-        auto [profileGuid, settings] = TerminalSettings::BuildSettings(_settings, newTerminalArgs, *_bindings);
```

Comment:
wow crazy how many times we were actually using the guid in the return value of this function too. Shame WinRT won't let us return the pair. Shame.

Reply:
I really don't love that this responsibility moved into TSM but ok.